src/main/flight/pid.h

```diff
@@ -324,6 +330,17 @@ typedef struct pidProfile_s {
     uint8_t yaw_type;                   // For wings: type of yaw (rudder or differential thrust)
     int16_t angle_pitch_offset;         // For wings: pitch offset for angle modes; in decidegrees; positive values tilting the wing down
 
+    // use aerodynamics
+    uint8_t  ad_mode;                   //For wings: The aerodynamics using mode: off, angle of attack, tpa
+    uint8_t ad_zero_lift_c;            //For wings: aerodynamics lift force coefficient in case of zero angle of attack value, 1000*Clift0 units
+    uint8_t ad_differ_lift_c;          //For wings: aerodynamics lift force coefficient differencial by angle of attack value, 1000*dClift/dAoA units, AoA [grad]
+    uint8_t ad_zero_drag_c;            //For wings: aerodynamics drag force coefficient in case of zero lift force value, 1000*Cdrag0 units
+    uint8_t ad_induce_drag_c;          //For wings: aerodynamics induce drag force coefficient, 1000*Cinduce units
+    uint16_t plane_mass;                //For wings: airplane mass, [g]
+    uint16_t wing_load;                 //For wings: wing load (mass / WingArea) [g/decimeter^2]
+    uint16_t air_density;               //For wings: the current atmosphere air density [mg/m^3], the MSA 1225 g/m^3 value is default. TODO: Dynamical air density computing by using baro sensors data
```

Comment:
maybe (it is all aerodynamics related):
```suggestion
    uint16_t ad_plane_mass;                //For wings: airplane mass, [g]
    uint16_t ad_wing_load;                 //For wings: wing load (mass / WingArea) [g/decimeter^2]
    uint16_t ad_air_density;               //For wings: the current atmosphere air density [g/m^3], the MSA 1225 g/m^3 value is default. TODO: Dynamical air density computing by using baro sensors data
```

wing loading should be in kg/m^2 to get a bit better resolution (common value can be 10g/dm^2 for trainers)

Reply:
The g/dm^2 simple to brain, maybe.
kg/m^2 is first*10,
thought about too...

Reply:
g/dm^2 won't work well for integer values, resolution is too low. It should be multiplied by 10 or 100 - kg/m^2 is SI and will add *10 factor

Reply:
Ok.